### DISCUSSION TITLE
Inline Build

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5198] [Unsupported Versions] [answered]
Hello all,

I would like to build my own SO device. I have a gen 7 i5 with 8 gig of RAM. I know I will probably have to up the RAM but wanted some input on the connectivity side. From my OPNsense firewall I have 3 internal LAN interfaces. My design is to directly connect those interfaces to a group of interfaces on my i5 build. I would have a second set of interfaces on the i5 build that would connect directly to my core switch, thus all traffic has to pass through the i5 build. 

Is this doable? 

Thanks,
Steve

**Comment by dougburks** [comment, marked as answer]
Sorry, we don't support running inline:
https://docs.securityonion.net/en/2.3/faq.html#can-security-onion-run-in-ips-mode
